src/configure/configure.ts

```diff
@@ -162,13 +162,14 @@ class Orchestrator {
 
         if (this.continueOrchestration) {
             await this.getSourceRepositoryDetails();
-            await this.getAzureSession();
+            this.inputs.azureSession = getAzureSession();
             let repoAnalysisResult = await this.getRepositoryAnalysis();
             await this.getSelectedPipeline(repoAnalysisResult);
 
             try {
                 if (!resourceNode) {
                     this.context['rightClickScenario'] = false;
+                    await this.getAzureSubscription();
```

Comment:
after subscription is selected we should update the `this.azureSession` to ensure that we have right token

Reply:
added condition